src/app/clusters/window-covering-server/window-covering-server.cpp

```diff
@@ -687,6 +687,17 @@ EmberAfStatus GetMotionLockStatus(chip::EndpointId endpoint)
     return EMBER_ZCL_STATUS_SUCCESS;
 }
 
+EmberAfStatus ExtractPercent100thsFromCommand(OPercent oPercent, OPercent100ths oPercent100ths, Percent100ths & percent100ths)
+{
+    if (oPercent.HasValue() || oPercent100ths.HasValue())
```

Comment:
Per "8.3.5. Element Requirements" in the device library spec, LiftPercentageValue and TiltPercentageValue have "!Matter" conformance, which means they must not be sent in Matter messages.  So why are we looking at them here?

Reply:
Same as previous answer 
@bzbarsky-apple can you confirm i must simply removed   <arg name="liftPercentageValue"  and   <arg name="tiltPercentageValue"  from the xml

Reply:
An also  remove the optional from <arg name="liftPercent100thsValue" type="Percent100ths" optional="true"/>

Reply:
well also looking at the device type overriding 
one can tell 
- you must support a liftPercent100thsValue for Matter
- you must support a liftPercentValue for ZigBee
but the cluster offer flexibility for both (need just 1) and there is nowhere  that say in device type
- you must not support liftPercentValue for Matter
- you must not support  liftPercent100thsValue for Zigbee

what do you recommend to do ?

Reply:
My understanding of the device type annotations is that they in fact say just that.  But please check with @CamWms to confirm.

Reply:
Since the IPR is out and Cam with the same statements i updated to simply use Percent100ths as mandatory @bzbarsky-apple